server/src/models/Podcast.js

```diff
@@ -0,0 +1,20 @@
+const Model = require("./Model");
+
+class Podcast extends Model {
+  static get tableName() {
+    return "podcasts";
+  }
+
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["name"],
+      properties: {
+        name: { type: "string", minLength: 1 },
+        description: { type: "string", minLength: 1 },
```

Comment:
if `description` is not required we shouldn't require a minimum length otherwise an error will be thrown that `"description should NOT be shorter than 1 character"`